server/testutils/factories/organisationAddress.ts

```diff
@@ -4,9 +4,10 @@ import { Factory } from 'fishery'
 import type { OrganisationAddress } from '@accredited-programmes/models'
 
 export default Factory.define<OrganisationAddress>(() => ({
-  addressLine1: faker.location.streetAddress(),
+  addressLine1: faker.helpers.arrayElement([null, faker.location.streetAddress()]),
```

Comment:
How likely is it that these address fields could be empty based on the `dev` data you've seen?

I wonder if for our general factory here we should assume the best, i.e. that these fields are present, and only set `null` values in specific test cases where we want to handle this null behaviour. Otherwise, any time we want to have a full address we'll need to override these fields.

Reply:
Of 128 active prisons, these are the `null` counts:
- 7 on `addressLine1`
- 83 on `addressLine2`
- 14 on `county`

So 5-65% depending on the field

Reply:
But yeah, I do wonder if it's best to keep the default predictable and happy?